lib/std/builtin.zig

```diff
@@ -98,6 +98,16 @@ pub const AtomicOrder = enum {
     SeqCst,
 };
 
+/// This data structure is used by the Zig language code generation and
+/// therefore must be kept in sync with the compiler implementation.
+pub const ReduceOp = enum {
+    And,
+    Or,
+    Xor,
+    Min,
+    Max,
```

Comment:
[lower case tag names](https://ziglang.org/documentation/master/#Style-Guide) for new enums please. I normally don't want to care about style stuff but I also don't want to have to make unnecessary breaking changes later.